lib/utils/session_vars.js

```diff
@@ -37,7 +37,7 @@ session_vars.get_branch_name = function () {
   /* Get the name of the repository's branch that is being tested */
   let branch_name = null;
   if ( process.env.BRANCH_PATH ) {
-      branch_name = process.env.BRANCH_PATH.replace( `refs/head/`, '' );
+      branch_name = process.env.BRANCH_PATH.replace( `refs/heads/`, '' );
```

Comment:
Also out of scope of this PR, but the `github.ref` variable in the action can be from [tags as well](https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables). DA doesn't let you pull from tags, just a note that should make it into docs or comments of the action.yml at some point.

Reply:
#530